crates/tauri-codegen/src/context.rs

```diff
@@ -309,7 +309,9 @@ pub fn context_codegen(data: ContextData) -> EmbeddedAssetsResult<TokenStream> {
     };
 
     if let Some(plist) = info_plist.as_dictionary_mut() {
-      if let Some(product_name) = &config.product_name {
+      if let Some(bundle_name) = &config.bundle.macos.bundle_name {
```

Comment:
same as the other one, you can use the same logic as below for CFBundleVersion. GitHub doesn't let me add a suggestion here for some reason tho.